contracts/Token.sol

```diff
@@ -23,7 +22,7 @@ import "./interfaces/IFeeSettings.sol";
  *
  * @dev The contract inherits from ERC2771Context in order to be usable with Gas Station Network (GSN) https://docs.opengsn.org/faq/troubleshooting.html#my-contract-is-using-openzeppelin-how-do-i-add-gsn-support and meta-transactions.
  */
-contract Token is ERC2771Context, ERC20, Pausable, AccessControl, Initializable {
+contract Token is ERC2771Context, ERC20PermitCloneable, Pausable, AccessControl {
```

Comment:
Do I understand correctly, that ERC2771 works with cloning, and only "Permit" (712) needs extra work?

Reply:
Yes, that is correct. This whole PR is only necessary if we still want to support Permit. 
We discussed the Pros and Cons of Permit a while ago, and this is definitely a big new item for the Contra list.
I would happily drop this PR if we decided together that still supporting Permit is not worth the additional risk anymore.